### PR TITLE
added an opentelemetry prometheus exporter and example config to the example backend

### PR DESCRIPTION
**Pull request by freben**
Bonus change: switch away from the deprecated `SingleHostDiscovery`

### CODE REVIEW

File: packages/backend/prometheus.yml [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+  scrape_interval: 15s
+  evaluation_interval: 15s
+
+scrape_configs:
+  - job_name: 'prometheus'
+    static_configs:
+      # This target address specifically works on Docker for MacOS - you may
+      # need to change it to another host depending on your OS and Docker
+      # engine.
+      - targets: ['host.docker.internal:9464']
```
Comment (benjdlambert) [inline packages/backend/prometheus.yml:21]: is this gonna throw a load of errors if docker isn't running or just one?

**Review by benjdlambert — APPROVED** [review]
We should probably add some more official docs around this setup too, but not needed for this PR I think

File: packages/backend/prometheus.yml (hunk 1 above)
Comment (freben) [reply packages/backend/prometheus.yml:21]: Actually none at all. If I intentionally add a typo in the host name it just silently does nothing at all.

**Comment by freben** [comment]
Regarding docs, I think we should probably have a module for the new backend system that you just add and you're done.

**Pull request merged**